02-express-tutorial/public/index.html

```diff
@@ -0,0 +1,13 @@
+const express = require('express')
```

Comment:
This is the app.js file.  Try adding some html instead so when you go to localhost:3000 you will see it load